### ISSUE TITLE
Using Bookdown to compile a FarmOS manual 

### ISSUE THREAD

**Issue by MorgsJohn** [farmOS/farmOS#149]
# Question or Suggestion

## Following on the monthly call (13/3/29).

We spoke about finding ways to encourage or open community participation in the continued development of FarmOS. One of the suggestions was to have places to point people for contributing and learning. In my time learning R I used a number of different books written in RMarkdown using Bookdown to help me manipulate my own coding. I think that there is a lot of content on the current website that would fit nicely into a  #bookdown format. I have attached links to a few examples if you are not familiar with the format. 

https://happygitwithr.com/
https://bookdown.org/yihui/bookdown/
https://bookdown.org/yihui/rmarkdown/
https://bookdown.org/

PS, I can't find the **tags** function or know the markdown to make it *pretty*

**Comment by mstenta** [comment]
Cool thanks for sharing @MorgsJohn ! Worth noting that farmOS.org (where the user guide lives) is all written in Markdown already, and we use a package called [Mkdocs](https://www.mkdocs.org/) to build it into HTML. So maybe similar to what you're suggesting?

For example:

https://raw.githubusercontent.com/farmOS/farmOS.org/7.x-1.x/docs/guide/areas.md

Automatically becomes:

https://farmos.org/guide/areas/

**Comment by MorgsBrew** [comment]
Yes. It is similar. I have avoided getting into markdown as I started in R. I like that RMarkdown has simplified the amount of pure coding that needs to get typed for the production of documents. This does not imply it is easier to learn. I suppose there is no easy way to get people learning code to improve the user manuals.

**Comment by mstenta** [comment]
I'm going to transfer this over to the farmOS.org issue queue, since that's where the user guide currently resides.
